catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrFilterDelegateTest.java

```diff
@@ -473,9 +473,9 @@ public void testXpathOR() {
         String combinedExpectedFilter =
                 "{!xpath}xpath:\"(" + xpath + "[contains(lower-case(.), 'example1')] or " + xpath
                         + "[contains(lower-case(.), 'example2')])\"";
-        String expectedIndex =
-                "{!xpath}(xpath_index:\"" + xpath + "[contains(lower-case(.), 'example1')]\") OR "
-                        + "(xpath_index:\"" + xpath + "[contains(lower-case(.), 'example2')]\")";
+        //        String expectedIndex =
```

Comment:
Let's delete both. If someone wants to look at it later, they can always use version control.